src/components/home/employees/EmployeeForm.js

```diff
@@ -63,7 +59,7 @@ class EmployeeForm extends React.Component {
       confirmButtonColor: '#3085d6',
```

Comment:
Create a utility for sweetalert like the toast util that we have. Just a wrapper around sweetalert